internals/getSecurityHeaders.js

```diff
@@ -64,9 +64,7 @@ const getCsp = ({ isExtension, isDev }) =>
     frame-ancestors
       ${isExtension ? dappFrameAncestors : `'self'`};
     frame-src
-      'self'
-      https://global.transak.com
```

Comment:
we will need ping devops during a next release https://github.com/search?q=repo%3Aoasisprotocol%2Finternal-ops%20transak&type=code

Reply:
yeah, although not critical if we keep running old headers for a while